python/setup.py

```diff
@@ -29,8 +28,14 @@
 
 
 def build_models(target_dir):
-    CmdStanPyBackend.build_model(target_dir, MODEL_DIR)
-    PyStanBackend.build_model(target_dir, MODEL_DIR)
+    backends = get_backends_from_env()
```

Comment:
To maintain current functionality, we want this to default to pystan if no env variables are specified.

Reply:
This is exactly how it behaves https://github.com/facebook/prophet/pull/1083/files#diff-85ff8370a138a633c3303428fb5eb0fcR12